components/calendar/src/julian.rs

```diff
@@ -153,6 +154,10 @@ impl Calendar for Julian {
     fn debug_name(&self) -> &'static str {
         "Julian"
     }
+
+    fn any_calendar_kind(&self) -> Option<AnyCalendarKind> {
+        None
```

Comment:
Question: why `None`?

Reply:
We don't support Julian within AnyCalendar since it's not (yet) formattable. We coudl if we reserved a key for it but it would be untestable without data.